newswires/client/src/App.tsx

```diff
@@ -217,15 +219,30 @@ export function App() {
 											>
 												Newswires
 											</EuiLink>
-											<EuiBetaBadge
-												label="Under construction"
-												title="Currently under construction"
-												color={'accent'}
-												size="s"
-												css={css`
-													margin-left: 8px;
-												`}
-											/>
+											<EuiShowFor sizes={['xs', 's']}>
+												<EuiBetaBadge
+													iconType={'beaker'}
+													label="Currently under construction"
+													aria-label="(Under construction)"
```

Comment:
aria-label is required here because for some reason the label & title props don't get put into the accessibility tree otherwise, so will be missing from the heading text for screen readers. 

I also added the same aria-label to the badge for larger screens below (although the label does get included in the accessibility tree there because it's rendered as visible text on the page) so that the heading is standardised across the breakpoints